bigtable/examples/run_examples_utils.sh

```diff
@@ -19,6 +19,47 @@ readonly CBT_CMD="${CBT:-${GOPATH}/bin/cbt}"
 readonly CBT_EMULATOR_CMD="${CBT_EMULATOR:-${GOPATH}/bin/emulator}"
 readonly CBT_INSTANCE_ADMIN_EMULATOR_CMD="../tests/instance_admin_emulator"
 
+function cleanup_instance {
+  local project=$1
+  local instance=$2
+  shift 2
+
+  if [ -z "${project}" -o -z "${instance}" ]; then
+    return
+  fi
+
+  echo
+  echo "Cleaning up test instance projects/${project}/instances/${instance}"
+  local setenv="env"
+  if [ -n "${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST:-}" ]; then
+    setenv="env BIGTABLE_EMULATOR_HOST=${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST}"
+  fi
+  ${setenv} ../examples/bigtable_samples_instance_admin delete-instance "${project}" "${instance}"
+}
+
+function exit_handler {
+  local project=$1
+  local instance=$2
+  shift 2
+
+  cleanup_instance "${project}" "${instance}"
+  if [ -n "${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST:-}" ]; then
+    # If the test is running against the emulator there is no need to cleanup
+    # the instance, just kill the emulators.
+    kill_emulators
```

Comment:
How about adding:

```bash
echo "Stopping Cloud Bigtable emulator...
kill_emulators
echo "Done."
```

Reply:
`kill_emulators` already prints what it is doing, but I like the idea.